core/integration/features/liquidity-provision/verify-CcySiska.feature

```diff
@@ -241,39 +225,12 @@ Feature: check the impact from change of market parameter: market.liquidity.stak
     #check the volume on the order book
     Then the order book should have the following volumes for market "ETH/MAR22":
       | side | price | volume |
-      | sell | 1100  | 1      |
+      | sell | 1100  | 10001  |
       | sell | 1014  | 0      |
       | sell | 1010  | 1      |
       | sell | 1000  | 0      |
       | buy  | 1000  | 0      |
       | buy  | 990   | 1      |
       | buy  | 986   | 0      |
-      | buy  | 900   | 1      |
-
-    #check position (party0 has no position)
-    Then the parties should have the following profit and loss:
```

Comment:
the remainder of the test has nothing to do with checking impact of `market.liquidity.stakeToCcyVolume` parameter